libnetwork/netutils/utils_linux_test.go

```diff
@@ -202,6 +202,14 @@ func TestGenerateRandomName(t *testing.T) {
 	if name1 == name2 {
 		t.Fatalf("Expected differing values but received %s and %s", name1, name2)
 	}
+
+	defer func() {
+		r := recover()
+		if r == nil {
+			t.Fatal("A panic should occur with size greater than 64")
+		}
+	}()
+	GenerateRandomName("veth", 65)
```

Comment:
This isn't an ideal way to test for a panic. I can create a test util that tests for panic using similar logic so that way more tests can be appended.

Reply:
The panic has been removed, so this comment is no longer valid.